enterprise/reporting/error_reporting.go

```diff
@@ -453,8 +459,18 @@ func (edr *ErrorDetailReporter) mainLoop(ctx context.Context, c types.SyncerConf
 					deletedRows += len(reports)
 				}
 				// vacuum error_reports_details table
+				var sizeEstimate int64
+				if err := dbHandle.QueryRowContext(
+					ctx,
+					`SELECT pg_table_size(oid) from pg_class where relname = $1`, ErrorDetailReportsTable,
```

Comment:
Shall we run this query only if the deleted rows condition fails? If it succeeds there is no need to run it.